pkg/filters/filters.go

```diff
@@ -74,16 +74,7 @@ func FiltersFromRequest(r *http.Request) ([]string, error) {
 		return nil, err
 	}
 
-	libpodFilters := make([]string, 0, len(filters))
-	for filterKey, filterSlice := range filters {
-		f := filterKey
-		for _, filterValue := range filterSlice {
-			f += "=" + filterValue
-		}
-		libpodFilters = append(libpodFilters, f)
-	}
-
-	return libpodFilters, nil
+	return getLibpodFilters(filters), nil
```

Comment:
Hum… on second thought, it would be better to unit-test at least the full parsing step, starting with the `raw` value; or, perhaps even, the whole `FiltersFromRequest` function, including a simulated `Request` with real-life strings in `Query`

Reply:
Fixed